Packages/com.unity.multiplayer.samples.coop/Utilities/Net/UnityRelayUtilities.cs

```diff
@@ -1,82 +0,0 @@
-using System;
```

Comment:
We should similarly add change notes inside of the package [changelog](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/blob/main/Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md) that we removed Relay dependencies.
We can leave it as unreleased for now, and upgrade the package version along with the next Boss Room release.